.circleci/config.yml

```diff
@@ -32,15 +32,15 @@ jobs:
           name: Install napari-dev
           command: |
             . venv/bin/activate
-            python -m pip install -e "napari/[pyside,dev]"
+            python -m pip install -e "napari/[pyqt5,docs]"
```

Comment:
Do you still need dev as well?

Reply:
I don't think so?
Honestly, we have so many dependency groups that I'm not sure anymore!